scripts/build-on-travis.sh

```diff
@@ -43,6 +43,16 @@ else
   if test "x$RUN_LINT" = "xyes"; then
     llvm-config --version
     clang++ --version
+
+    # Run linter for x86 target
+    time make --jobs $JOBS lint
+
+    # Run linter for p and z targets
+    export TARGET_ARCH=p
+    export TARGET_BITS=64
+    time make --jobs $JOBS lint
+
+    export TARGET_ARCH=z
```

Comment:
I guess you're "inheriting" the earlier setting of TARGET_BITS here, but I would prefer you set it explicitly so that it's clear what it is being tested  for the z platform and also will be less brittle to subsequent changes.